frontend/src/components/website/adding_master/StepAddDomain.vue

```diff
@@ -59,7 +59,11 @@
     export default {
         name: 'StepAddDomain',
         props: {
-            errorMessage: String,
+            errorMessage: {
+                type: String,
+                required: false,
```

Comment:
not a blocker, but required not required when you have default key)) (not a blocker, just something to remember for future)